### DISCUSSION TITLE
Version 7.44.0-rc.0

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#10084] [General]
📄 Form Component support custom fetcher with `FormData` and `Json`

```tsx
<Form
  action="/api"
  fetcher={async (
    action,
    { method, values, event, formData, formDataJson }
  ) => {
    await fetch(action, { body: formData });
    // or with JSON
    await fetch(action, {
      header: { "content-type": "application/json" },
      body: formDataJson,
    });
  }}
/>;
```

📄 `useForm` support `TTransformedValues`
```tsx
UseFormReturn<TFieldValues, TContext, TTransformedValues>
```

🩴 close #10078 to prevent stabled async validation (#10082)
🐞 fix #10064 native validation when subscribing to isValid (#10072)
📝 correct typo in field array type declaration (#10066)

thanks to @chrisbarless

<hr /><em>This discussion was created from the release <a href='https://github.com/react-hook-form/react-hook-form/releases/tag/v7.44.0-rc.0'>Version 7.44.0-rc.0</a>.</em>

**Comment by rodrigoreeis** [comment]
I'm still in doubt what would be the great benefit of being able to fetch inside the form?

**Reply by bluebill1049** [reply]
Hi @rodrigoreeis have you seen this RFC: https://github.com/react-hook-form/react-hook-form/discussions/9856 have a look at it and let's discuss on this.
